cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Protos/cmsAdapter.proto

```diff
@@ -282,6 +286,8 @@ message CaseDetail {
     string name = 18;
     string driverLicenseNumber = 19;
     google.protobuf.Timestamp birthDate = 20;
+    string firstName = 21;
+    string surname = 22;
```

Comment:
driver portal view model uses "LastName". Are we using "Surname" anywhere else?

Reply:
driver portal view model is not being used, so I think leaving it "Surname" is fine